src/menus/ViewMenus.cpp

```diff
@@ -328,12 +327,6 @@ auto ViewMenu()
                   TimeSelectedFlag(), wxT("Ctrl+E") ),
                Command( wxT("ZoomToggle"), XXO("Zoom &Toggle"), OnZoomToggle,
                   TracksExistFlag(), wxT("Shift+Z") )
-            ),
-            Section( "",
-               Command( wxT("AdvancedVZoom"), XXO("Advanced &Vertical Zooming"),
-                  OnAdvancedVZoom, AlwaysEnabledFlag,
-                  Options{}.CheckTest( wxT("/GUI/VerticalZooming"), false ) )
-            )
```

Comment:
This leaves the function `OnAdvancedVZoom` unused, so delete it too

Reply:
Did you scan the code for all occurrences of "/GUI/VerticalZooming"?

Maybe I should review #5685 first.

So far this looks like a change I could not approve, because of incompleteness.  The code is still governed by the behavior of a preference that you can't change any more in the user interface.  That behavior would be "stuck" one way or another depending on your audacity.cfg file.

By the way -- the long-distance coincidence of repeated string literals for the preference pathname  is a thing I have long disliked.  So instead there are the classes in `Prefs.h` -- `BoolSetting`, etc. -- which ought to eliminate all of these repetitions some day as a code cleanup.  There would be a lot of volume of work there.

Maybe you can figure out examples and volunteer to do some of those cleanups if you like.  That's for preferences we don't remove.